Description: Fix cleanup rule in Makefile
 also remove fsogsm.h during make clean
Author: Rico Rommel <rico@bierrommel.de>
--- a/src/lib/Makefile.am
+++ b/src/lib/Makefile.am
@@ -137,6 +137,7 @@
 CLEANFILES = \
 	$(BUILT_SOURCES) \
         $(libfsogsm_la_VALASOURCES:.vala=.c) \
+	fsogsm.h \
 	$(NULL)
 
 MAINTAINERCLEANFILES = \
